index.js

```diff
@@ -334,6 +339,108 @@ class LiteApi {
             "status": "success",
             "data": data.data
         }
+    }
+     /**
+    * The API returns the list of hotel facilities available in the system.
+    * @returns {array} - The result of the operation.
+    */
+     async getHotelFacilities() {
+        let errors = [];
```

Comment:
These lines are useless. You are creating an array of errors which is never going to be used.